cmd/machine-config-operator/bootstrap.go

```diff
@@ -63,6 +64,7 @@ func init() {
 	bootstrapCmd.MarkFlagRequired("etcd-image")
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.kubeClientAgentImage, "kube-client-agent-image", "", "Image for Kube Client Agent.")
 	bootstrapCmd.MarkFlagRequired("kube-client-agent-image")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.clusterEtcdOperatorImage, "cluster-etcd-operator-image", "", "Image for Cluster Etcd Operator. An empty string here means the cluster boots without CEO.")
```

Comment:
i think you will need to update installer to pass this in?

Reply:
see https://github.com/openshift/installer/pull/1847

Reply:
yep we are just working through the maze here. My hopes are that because it is not a required flag we can sneak in it into MCO before the installer.

Reply:
looking at my past images stuff: without required should be ok!